draft-ietf-quic-multipath.md

```diff
@@ -351,31 +314,41 @@ does not discuss when a client decides to initiate a new path. We
 delegate such discussion in separate documents.
 
 To let the peer open a new path, an endpoint needs to provide its peer with connection IDs
-and associated Path Identifiers for the new path.
-
-To open a new path, an endpoint SHALL use different Connection IDs on different paths.
-Still, the receiver may observe the same Connection ID used on different
-4-tuples due to, e.g., NAT rebinding. In such case, the receiver reacts
+for at least one unused path identifier. Endpoints SHOULD use the MP_NEW_CONNECTION_ID
+frame to provide new connection IDs and, respectively, the MP_RETIRE_CONNECTION_ID frame to
+retire connection IDs after a successful handshake indicating multipath support by both endpoints.
+
+Endpoints MUST NOT issue connection IDs with path identifiers larger than
+the path limitation advertised by the peer, corresponding to the maximum value
+between the peer's initial_max_paths transport parameter and received
+MAX_PATHS frames.
+
+To open a new path, an endpoint MUST use a connection ID associated with
+a new, unused Path ID.
+Still, the receiver may observe a connection ID associated with a used Path ID
+on different 4-tuples due to, e.g., NAT rebinding. In such case, the receiver reacts
 as specified in {{Section 9.3 of QUIC-TRANSPORT}} by initiating path validation
-and using a new Connection ID for the same path ID.
+and using a new connection ID for the same Path ID.
 
 This proposal adds three multipath control frames for path management:
 
 - PATH_ABANDON frame for the receiver side to abandon the path
 (see {{path-abandon-frame}})
 - PATH_STANDBY and PATH_AVAILABLE frames to express a preference
-in path usage (see {{path-standby-frame}} and {{path-available-frame}}
+in path usage (see {{path-standby-frame}} and {{path-available-frame}})
 
 All new frames are sent in 1-RTT packets {{QUIC-TRANSPORT}}.
 
-## Path Initiation
+## Path Initiation {#path-initiation}
 
 Connection IDs cannot be reused, thus opening a new path requires the
```

Comment:
The logic here doesn't really work out anymore, since CIDs are now bound to a path.

Reply:
This is still correct. CID need still to be unique per connection. However, I agree that this half sentence is probably simply unnecessary.